### PR TITLE
Added Grab The carrot 

### LINKED ISSUE #4201: [New game]: Grab the carrot
### 🎮 Game Request

Game logic and basic description

it is basically a game like temple run where the character will be running and behind us there will be a monster chasing and we need jump the obstacle and other stuffs .

image

### Point down the features

Game points

This will be warded to the user according to the user highscore 

pls do assign the GSSOC Label 

### Select program in which you are contributing

_No response_

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Aksshay88**
## PR Description 📜
I have included the Grab the carrot game where the user could play the game
Fixes #4201

<hr>
 
## Mark the task you have completed ✅

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [X] I have added screenshot for website preview in assets/images 
- [X] I have added entries for my game in main README.md
- [X] I have added README.md in my folder 
- [X] I have added working video of the game in README.md (optional)
- [X] I have specified the respective issue number for which I have requested the new game.

<hr>

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Aksshay88 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by Aksshay88** [comment]
@kunjgit I have included the game You could review the PR and merge it .Thanks in advance

**Comment by Aksshay88** [comment]
@kunjgit I request you to review the PR once

File: assets/images/grab.png
Comment (ishita-43) [inline assets/images/grab.png:1]: Remove this if not needed or rename as your game name

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by Aksshay88** [comment]
@ishita-43 will remove it

**Comment by Aksshay88** [comment]
@ishita-43 I have removed it .You could check the pr now

**Review by ishita-43 — APPROVED** [review]
Appreciate it ☻.
Ready to merge!!

**Comment by kunjgit** [comment]
Thank you  @Aksshay88 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**